### ISSUE TITLE
Why Does It Immediately Sell After Buying?

### ISSUE THREAD

**Issue by wackytrader** [rckprtr/pumpdotfun-sdk#65]
Is it just to show how it works? Would it be difficult to execute a sell of all accts on demand (i.e., through a CLI)?

**Comment by saraspaudel** [comment]
Yes it is just to show how it works. You could adapt this SDK to build a node.js cli program that accepts any input (for example to sell multiple wallets)
